03week/towersOfHanoi.js

```diff
@@ -13,29 +13,67 @@ let stacks = {
   c: []
 };
 
+const stackA=stacks.a;
+const stackB=stacks.b;
+const stackC=stacks.c;
+const stacksArr=[stackA, stackB, stackC];
+// Create an array that houses all stacks so a forEach function can be applied(shortens code).
+
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
 function movePiece() {
-  // Your code here
+
+     let startStack.pop();
+     let endStack.push(startStack);
+  //Write a function that allows you to move the blocks(pop/push).
+
+    }
+
+})
+
+});
+
+movePiece();
```

Comment:
This needs to be called in the towersOfHanoi function, and it should only be called if the move is legal